pages/in/settings/others.vue

```diff
@@ -64,22 +119,29 @@
 
 
 <script>
+/* eslint-disable no-param-reassign */
 import { mapActions, mapGetters } from 'vuex';
 
 import { logTrackerEvent } from '@/util/EventLogger';
+import CrossIcon from '@/assets/icons/cross.svg';
+import { isValidHttpUrl } from '@/util/ValidationHelper';
 
 import PoliciesLinks from '~/components/PoliciesLinks';
 
+const MAX_REDIRECT_URL_NUM = 5;
+
 export default {
   name: 'settings-others',
   components: {
     PoliciesLinks,
   },
   data() {
     return {
+      CrossIcon,
       isEmailEnabled: false,
       isEmailPreviouslyEnabled: false,
       isLoading: false,
+      viewModelPaymentRedirectWhitelist: [{ url: '', err: '' }],
```

Comment:
`viewModelPaymentRedirectWhitelist` -> `paymentRedirectURLWhitelist`

Reply:
Should I also make `actualPaymentRedirectWhitelist` -> `actualPaymentRedirectURLWhitelist`?